packages/event-consumer/kubernetes/dev/configmap.yaml

```diff
@@ -5,5 +5,5 @@ metadata:
   namespace: dev-refactor
 data:  
   KAFKA_CLIENT_ID: "dev-pagopa-interop-catalog"
-  KAFKA_GROUP_ID: "dev-pagopa-interop-catalog-consumers"
+  KAFKA_GROUP_ID: "catalog.public.event"
```

Comment:
to me this seems more a topic name that a consumer group.

BTW non-blocker right now, but it could be the right time to define naming conventions.

Reply:
However, I would change both the client_id and the group_id before merging.
The current values are not related to the consumer